dist/tools/ethos/ethos.c

```diff
@@ -519,13 +547,16 @@ int main(int argc, char *argv[])
 
     char ifname[IFNAMSIZ];
     strncpy(ifname, argv[1], IFNAMSIZ);
+#ifdef __linux__
```

Comment:
This `#ifdef` (as well as [this](https://github.com/RIOT-OS/RIOT/pull/6718/files#diff-33a73ae3a5a86becdcf02c036d777196R534) is unnecessary, as `flags` is ignored anyway for the non `__linux__` case.